requests/__init__.py

```diff
@@ -56,18 +56,37 @@
 except AssertionError:
     raise RuntimeError('Requests dependency \'urllib3\' must be version >= 1.21.1, < 1.22!')
 
+import warnings
+
+def get_version(package):
```

Comment:
If we are going to abstract this out to a function, maybe we add it to the urllib3 import to reduce the duplicated code to a single place?